the_mines/data/fussballdaten_2019_2020_1.html

```diff
@@ -0,0 +1 @@
ef="/bundesliga/1968/" tabindex=-1>1967/1968</a></li>\n<li><a href="/bundesliga/1967/" tabindex=-1>1966/1967</a></li>\n<li><a href="/bundesliga/1966/" tabindex=-1>1965/1966</a></li>\n<li><a href="/bundesliga/1965/" tabindex=-1>1964/1965</a></li>\n<li><a href="/bundesliga/1964/" tabindex=-1>1963/1964</a></li></ul></li>\n<li class=active><a href="/bundesliga/2020/23/">Spieltag</a></li>\n<li><a href="/bundesliga/tabelle/">Tabellen</a></li>\n<li><a href="/bundesliga/statistik/">Statistik</a></li>\n<li><a href="/bundesliga/spielplan/">Spielplan</a></li>\n<li><a href="/bundesliga/aktuelles/">Aktuelles</a></li>\n<li><a href="/bundesliga/historie/">Historie</a></li></ul></div></div><div class="box bs green-top" id=spiele data-speed-kit-id=id-spiele>\n<div class=kategorie-headline><h1><img class=vereinswappen src="https://storage.fussballdaten.de/source/1/wettbewerbhaupt/21/thumb/originale/thumb-bundesliga-2018.png" alt="">Bundesliga - 1. Spieltag</h1><div class=select-holder><select class="selectpicker my-picker change-url" data-size=7 data-live-search=true><option value=1 href="/bundesliga/2020/1/" selected data-tokens=1>1. Spieltag</option><option value=2 href="/bundesliga/2020/2/" data-tokens=2>2. Spieltag</option><option value=3 href="/bundesliga/2020/3/" data-tokens=3>3. Spieltag</option><option value=4 href="/bundesliga/2020/4/" data-tokens=4>4. Spieltag</option><option value=5 href="/bundesliga/2020/5/" data-tokens=5>5. Spieltag</option><option value=6 href="/bundesliga/2020/6/" data-tokens=6>6. Spieltag</option><option value=7 href="/bundesliga/2020/7/" data-tokens=7>7. Spieltag</option><option value=8 href="/bundesliga/2020/8/" data-tokens=8>8. Spieltag</option><option value=9 href="/bundesliga/2020/9/" data-tokens=9>9. Spieltag</option><option value=10 href="/bundesliga/2020/10/" data-tokens=10>10. Spieltag</option><option value=11 href="/bundesliga/2020/11/" data-tokens=11>11. Spieltag</option><option value=12 href="/bundesliga/2020/12/" data-tokens=12>12. Spieltag</option><option value=13 href="/bundesliga/2020/13/" data-tokens=13>13. Spieltag</option><option value=14 href="/bundesliga/2020/14/" data-tokens=14>14. Spieltag</option><option value=15 href="/bundesliga/2020/15/" data-tokens=15>15. Spieltag</option><option value=16 href="/bundesliga/2020/16/" data-tokens=16>16. Spieltag</option><option value=17 href="/bundesliga/2020/17/" data-tokens=17>17. Spieltag</option><option value=18 href="/bundesliga/2020/18/" data-tokens=18>18. Spieltag</option><option value=19 href="/bundesliga/2020/19/" data-tokens=19>19. Spieltag</option><option value=20 href="/bundesliga/2020/20/" data-tokens=20>20. Spieltag</option><option value=21 href="/bundesliga/2020/21/" data-tokens=21>21. Spieltag</option><option value=22 href="/bundesliga/2020/22/" data-tokens=22>22. Spieltag</option><option value=23 href="/bundesliga/2020/23/" data-tokens=23>23. Spieltag</option><option value=24 href="/bundesliga/2020/24/" data-tokens=24>24. Spieltag</option><option value=25 
```

Comment:
We probably don't want to have these `html` files sitting in our repo.  Keeping 1 or 2 abbreviated files in `tests/` may be a good idea for testing some of the features here.

Reply:
What do you mean by abbreviated?

Reply:
The file is very large, we could take out the parts we aren't interested in and just keep a few cases.

Reply:
Dunno if that's feasible. Open one and check it out - they are messy. They really aren't that large size-wise. If you can find a pain-free way to take out parts we don't need I'm all for it.